INTERNALS.md

```diff
@@ -211,3 +201,8 @@ All timestamps are represented as integers, milliseconds since the Unix epoch.
 
   Job identifiers are formatted as batch UIDs.
 
+`counters` (hash)
+
+  Telemetry counters.
```

Comment:
It seems a little confusing to rename `counters => gauges` and then have a new `counters` variable. Would it make more sense to have the telemetry stuff as `metrics` and leave the original alone?